resources/options.js

```diff
@@ -153,11 +156,27 @@ function switchOptions() {
     }
 }
 
+// Saves the default options by saving an object.
+function saveDefaults() {
+    for (var i = 0; i < options.length; i++) {
+        defaultOptions[options[i]] = optionsElement(i).prop('checked');
+    }
+}
+
+// Sets the options to the default options by updating the chrome storage and the options page.
+function restoreDefaults() {
+    chrome.storage.sync.set(defaultOptions, function() {});
```

Comment:
I get that :) 
As a user you can apply the same way of reasoning as for saving the options; "Hm.. did something change? Ah well, must have been saved. Lemme check.. Oeh, it -has- been saved."
@mpsijm @MathiasMeuleman what do you think? (as a voting)